.idea/libraries/Dart_Packages.xml

```diff
@@ -5,182 +5,182 @@
         <entry key="async">
           <value>
             <list>
-              <option value="$PROJECT_DIR$/../../NeilLab/flutter/.pub-cache/hosted/pub.dartlang.org/async-2.0.8/lib" />
+              <option value="C:/flutter/.pub-cache/hosted/pub.dartlang.org/async-2.0.8/lib" />
```

Comment:
It's my bad but could you help to fix `git ignore` to ignore all `.idea` changes?

Reply:
Sure thing, I'll thought about removing it too

Reply:
👍